installing/installing_aws/installing-aws-network-customizations.adoc

```diff
@@ -68,7 +68,7 @@ include::modules/configuring-hybrid-ovnkubernetes.adoc[leveloffset=+1]
 
 [NOTE]
 ====
-For more information on using Linux and Windows nodes in the same cluster, see Understanding Windows container workloads.
+For more information on using Linux and Windows nodes in the same cluster, see xref:../../windows_containers/understanding-windows-container-workloads.adoc#understanding-windows-container-workloads[Understanding Windows container workloads].
```

Comment:
Don't we also need this note for https://github.com/openshift/openshift-docs/blob/main/installing/installing_vsphere/installing-vsphere-network-customizations.adoc?